src/components/ScrollToTop/__tests__/ScrollToTop.test.tsx

```diff
@@ -0,0 +1,30 @@
+import ScrollToTop from "../ScrollToTop";
+import getDesignTokens from "../../../theme/theme";
+import { scroolMethods } from "../../../Utils/scroolToTop";
+import { ThemeProvider } from "@emotion/react"
+import { IconButton } from "@mui/material";
+
+const theme = getDesignTokens('light')
+
+describe("Testando o componente ScrollToTop", () => {
+    it("Componente renderiza normalmente quando houver scroll na tela (chamar a função cy.mount com algum outro conteúdo com altura para que haja espaço para scroll", () => {
+        cy.window().then((win) => {
+            win.document.documentElement.scrollTop = 1000
+            win.document.body.scrollTop = 1000
+        })
+        cy.mount(<ThemeProvider theme={theme}><ScrollToTop data-class-ref='button' /></ThemeProvider>)
+        cy.get("[data-cy='scrollToTop']").should('exist')
```

Comment:
Mudar asserção para be.visible, ScrollToTop sempre é renderizado na DOM, mas só deve ser vísivel após scroll